.github/workflows/example-local.yml

```diff
@@ -3,23 +3,34 @@ name: Create a provenance
 on:
   workflow_dispatch:
     inputs:
-      text_input:
+      ingredient_input:
         description: 'Ingredient to input in the artifact text file'     
         required: false
+      car_input:
+        description: 'Car to input in the artifact text file'     
+        required: false
 
 jobs:
   build:
     name: Build artifact
     runs-on: ubuntu-latest
     steps:
       # traditionally you would build your code here and generate an artifact
-      - name: Create artifact
+      - name: Create first artifact
         run:  echo "onion, tomato, jalapeno, cilantro, lime, salt" > salsa.txt
 
+      - name: Create second artifact
+        run:  echo "audi, tesla, bmw" > cars.txt
+
       - name: Input workflow ingredient
-        if:  ${{ github.event.inputs.text_input != '' }}
+        if:  ${{ github.event.inputs.ingredient_input != '' }}
+        run: |
+          echo ", ${{github.event.inputs.ingredient_input}}" >> salsa.txt
+
+      - name: Input workflow cars
+        if:  ${{ github.event.inputs.car_input != '' }}
         run: |
-          echo ", ${{github.event.inputs.text_input}}" >> salsa.txt
+          echo ", ${{github.event.inputs.car_input}}" >> cars.txt
```

Comment:
You should also upload the cars artifact

Reply:
Fixed